src/scss/fields.scss

```diff
@@ -1,98 +1,63 @@
-@include oFormsClass(o-forms__field-group) {
+@if not $o-forms-is-silent {
+.o-forms-group {
```

Comment:
If your adding the properties directly to the classes when not in silent mode, how are they used outside of silent mode?